lib/mobility-core/src/Kernel/External/GoogleTranslate/Client.hs

```diff
@@ -25,17 +28,23 @@ import Servant.Client.Core (ClientError)
 
 translate ::
   ( CoreMetrics m,
-    MonadFlow m
+    MonadFlow m,
+    HasFlowEnv m r '["kafkaProducerTools" ::: KafkaProducerTools],
+    HasFlowEnv m r '["appPrefix" ::: Text]
   ) =>
   BaseUrl ->
   Text ->
   Text ->
   Text ->
   Text ->
+  Text ->
   m GoogleTranslate.TranslateResp
-translate url apiKey source target query = do
-  callAPI url (API.translate apiKey source target query) "translate"
-    >>= checkGoogleTranslateError url
+translate url apiKey source target query someId = do
+  res <-
+    callAPI url (API.translate apiKey source target query) "translate"
+      >>= checkGoogleTranslateError url
+  streamToKafka (GoogleTranslate.TranslateData {request = GoogleTranslate.TranslateReq {..}, response = res}) someId "google-translate-data"
```

Comment:
Could you remake all these changes to not change any low level interface implementations? I'm pretty sure that you can just call this func in `Kernel.External.Maps.Interface` (or in complitely new interface wrapper for these additional kind of things `Kernel.External.ExtendedMaps.Interface`, but if'm not sure if we need whole new wrapper here)